### PR TITLE
update the build link

### PR DESCRIPTION
**Pull request by sujinmkang**
removed the wrong platform build links and updated the build link with sonic-swss-common-build

**What I did**
removed the wrong platform build links and updated the build link with sonic-swss-common-build
**Why I did it**
the build link is not existed
**How I verified it**
verified if there is no broken images for build icon
**Details if related**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
-Broadcom[![Broadcom](https://sonic-jenkins.westus2.cloudapp.azure.com/job/broadcom/job/sonic-swss-build/badge/icon)](https://sonic-jenkins.westus2.cloudapp.azure.com/job/broadcom/job/sonic-swss-build/)
-Mellanox[![Mellanox](https://sonic-jenkins.westus2.cloudapp.azure.com/job/mellanox/job/sonic-swss-build/badge/icon)](https://sonic-jenkins.westus2.cloudapp.azure.com/job/mellanox/job/sonic-swss-build/)
-VS[![VS](https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/sonic-swss-build/badge/icon)](https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/sonic-swss-build/)
```
Comment (qiluo-msft) [inline README.md]: >https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/sonic-swss-build/ [](start = 9, length = 77)

I think the vs icon is still working and useful.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,4 @@
...
+sonic-swss-common-build[![sonic-swss-common-build](https://sonic-jenkins.westus2.cloudapp.azure.com/job/common/job/sonic-swss-common-build/badge/icon)](https://https://sonic-jenkins.westus2.cloudapp.azure.com/job/common/job/sonic-swss-common-build/)
```
Comment (qiluo-msft) [inline README.md]: >sonic-swss-common-build [](start = 0, length = 23)

swss-common icon is not related.

File: README.md (hunk 1 above)
Comment (sujinmkang) [reply README.md]: done!

File: README.md [hunk 3]
```diff
@@ -1,6 +1,4 @@
-Broadcom[![Broadcom](https://sonic-jenkins.westus2.cloudapp.azure.com/job/broadcom/job/sonic-swss-build/badge/icon)](https://sonic-jenkins.westus2.cloudapp.azure.com/job/broadcom/job/sonic-swss-build/)
-Mellanox[![Mellanox](https://sonic-jenkins.westus2.cloudapp.azure.com/job/mellanox/job/sonic-swss-build/badge/icon)](https://sonic-jenkins.westus2.cloudapp.azure.com/job/mellanox/job/sonic-swss-build/)
-VS[![VS](https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/sonic-swss-build/badge/icon)](https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/sonic-swss-build/)
+VS[![VS](https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/sonic-swss-build/badge/icon)](https://https://sonic-jenkins.westus2.cloudapp.azure.com/job/vs/job/sonic-swss-build/)
```
Comment (qiluo-msft) [inline README.md]: >https://https:// [](start = 99, length = 16)

why double prefix?

Comment (sujinmkang) [reply README.md]: done

**Comment by daall** [comment]
retest this please

**Comment by sujinmkang** [comment]
retest this please

**Comment by daall** [comment]
retest this please

**Comment by daall** [comment]
retest this please

**Pull request merged**